src/button/styles.scss

```
@@ -147,6 +147,19 @@
     margin-inline-start: awsui.$space-xxs;
   }
 
+  &.variant-inline-link {
+    > .icon-left {
+      inset-inline-start: 0;
+      margin-inline-end: awsui.$space-xs;
+    }
+
+    > .icon-right {
+      position: relative;
+      inset-inline-end: 0;
+      margin-inline-start: awsui.$space-xs;
+    }
```

```suggestion
    > .icon-right {
      inset-inline-end: 0;
      margin-inline-start: awsui.$space-xs;
    }
```

The position property is not needed - it's inherited from https://github.com/cloudscape-design/components/pull/2081/files#diff-6529fe18ff5331f59396707323f6469db1d2002399ab5e95515ab2c43164889aR144

---

Thanks, removed.